static-site/app/groups/[[...groups]]/individual-group-page.tsx

```diff
@@ -211,11 +211,17 @@ export default function IndividualGroupPage({
           <ScrollableAnchor id={"datasets"}>
             <div>
               <h3 className="centered">Available datasets</h3>
-              <ListResources
-                resourceType="dataset"
-                versioned={false}
-                fetchResourceGroups={resourcesCallback}
-              />
+              {datasets.length === 0 ? (
+                <h4 className="centered">
+                  No datasets are available for this group.
+                </h4>
+              ) : (
+                <ListResources
+                  resourceType="dataset"
+                  versioned={false}
+                  fetchResourceGroups={resourcesCallback}
+                />
+              )}
```

Comment:
[comment after reading the diff]

This seems like it would be more generally useful and thus better placed within `<ListResources/>`